### ISSUE TITLE
JATS: Decoding of (eLife) affiliations

### ISSUE THREAD

**Issue by FAtherden-eLife** [stencila/encoda#458]
Not all of the content in an affiliation is decoded. eLife's affiliations contain 1 or 2 institution elements, and it appears that only the first one is decoded. 

For example, the command:
```
encoda convert https://elifesciences.org/articles/46842v2 46842.jsonld
```

produces a file with:
```json
 "authors": [
    {
      "givenNames": [
        "Florian"
      ],
      "familyNames": [
        "Thüroff"
      ],
      "type": "Person",
      "affiliations": [
        {
          "name": "Arnold Sommerfeld Center for Theoretical Physics and Center for NanoScience, Department of Physics",
          "address": "Munich, Germany",
          "type": "Organization"
        }
      ]
    },  ...
```

The original affiliation input in the XML is:
```xml
<aff id="aff1">
     <label>1</label>
     <institution content-type="dept">Arnold Sommerfeld Center for Theoretical Physics and Center for NanoScience, Department of Physics</institution>
     <institution>Ludwig-Maximilians-Universität München</institution>
     <addr-line><named-content content-type="city">Munich</named-content></addr-line>
     <country>Germany</country>
</aff>
```

In this instance the content in the second `institution`, Ludwig-Maximilians-Universität München, is not decoded.

eLife content will always contain 1 or two institutions. (For reference, other publishers may include many more institution elements).

**Comment by nokome** [comment]
The Stencila schema `Organization` nodes have the property [`parentOrganization`](https://schema.org/parentOrganization).

Given that, I propose to decode JATS `<institution>` elements assuming that the  each successive element is a parent of the previous. This will be the easiest structure for encoding to HTML. The above example would then be decoded as 

```json
{
    "type": "Organization",
    "name": "Arnold Sommerfeld Center for Theoretical Physics and Center for NanoScience, Department of Physics",
    "parentOrganization": {
        "type": "Organization",
        "name": "Ludwig-Maximilians-Universität München",
        "address": "Munich, Germany"
    }
}
```

It appears that the values in`content-type` are not standardized but that "dept" is a convention for department https://jats.nlm.nih.gov/archiving/tag-library/1.3d1/element/institution.html. We could use the [`departments`](https://schema.org/department) property, but that would result in variation in the structure which would be more difficult to handle when encoding to other formats such as HTML.

Also we should consider adding https://schema.org/PostalAddress to the schema to better capture address information.

**Comment by nokome** [comment]
This has been fixed in de8abb6f7654ca70810be6f667020d95563d0b1c. Contrary to the above suggestion, I decided that the address was best attached to the `Organization`, rather than the `parentOrganization` (e.g a laboratory, may have a different address to the university it belongs to, so the address is probably attached to the lab). 

Decoding of addresses to the schema.org vocabulary is also included in that commit using this mapping:

```ts
    const mapping: Record<string, string> = {
      'addr-line': 'streetAddress',
      city: 'addressLocality',
      state: 'addressRegion',
      country: 'addressCountry',
      'postal-code': 'postalCode'
    }
```

An example from an eLife article (in YAML):

```yaml
  - type: Person
    affiliations:
      - type: Organization
        address:
          type: PostalAddress
          addressCountry: United Kingdom
          addressLocality: Cambridge
        name: Department of Chemistry
        parentOrganization:
          type: Organization
          name: University of Cambridge
```
